src/components/Insight/Comments.js

```diff
@@ -7,34 +7,39 @@ import toReact from 'svelte-adapter/react'
 import SvelteComments from 'insights-app/lib/components/comments/Comments'
 import { client } from '../../index.js'
 import {
```

Comment:
**[D-0-0]** Since this component is used as a generic interface for working with comments, `src/components/Insight/Comments.js` component should be abstracted away from any target realisation, i.e no default arguments related to the specific case. `editComment`, `deleteComment`,`createComment`,`getComments` should be passed only as the props when using it. By doing that, component's internal design will be significantly improved